### DISCUSSION TITLE
Sort by cell type in Step 4 heatmaps

### DISCUSSION THREAD

**Discussion by Gorman222** [aertslab/SCENIC#272] [R]
Is there a way to sort the binary regulon activity heatmaps created in Step 4 by the cellInfo and colVars files so that a single cell type would be a "block" instead of being somewhat dispersed?
Thanks.

**Comment by EhsanRz** [comment]
What happened to the answer to this question? The same thing is happening to me? Is it necessary to sort the cells before making scenic objects?
